### PR TITLE
docs: Add notice regarding fault proofs coming to mainnet on Base

### PR DESCRIPTION
**Pull request by wbnns**
**What changed? Why?**

Adds a notice to docs.base.org for fault proofs going live with helpful information

**Notes to reviewers**

N/A

**How has it been tested?**

Localhost

### CODE REVIEW

**Review by brendan-defi — APPROVED** [review]

File: apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md [hunk 1]
```diff
@@ -10,7 +10,6 @@ keywords:
     Permissionless output proposals,
     Permissionless challenges,
     Withdrawals,
-    Base Sepolia testnet,
```
Comment (anikaraghu) [inline apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md:13]: should we add Mainnet or is that covered with `Base` above?

File: apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Preparing for fault proofs on Base Sepolia
-slug: /preparing-for-fault-proofs-on-base-sepolia
-description: Fault proofs are expected to go live for Base Sepolia testnet in mid-July.
+title: Preparing for fault proofs on Base
+slug: /preparing-for-fault-proofs-on-base
+description: Fault proofs are expected to go live on Base Mainnet in late October / early November.
```
Comment (anikaraghu) [inline apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md:4]: let's update all instances now to just `late October`

File: apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md [hunk 3]
```diff
@@ -46,25 +45,23 @@ These changes allow anyone to withdraw funds from Base to L1 without having to r
...
 
-**If your team is operating a bridge on Base Sepolia:**
+If your withdrawal of funds from Base to Ethereum coincides with the upgrade proceeding in late October / early November, you will be required to resubmit your withdrawal.
 
-- Please provide your users with a notice on your UI to inform them that fault proofs will be enabled for Base Sepolia in mid-July.
-- Please ensure it’s clear to your users that testnet withdrawals will now take at least seven days.
+**If your team is operating a bridge on Base Mainnet:**
+
+- Please provide your users with a notice on your UI to inform them that fault proofs will be enabled for Base Mainnet in late October / early November.
+- Please ensure it’s clear to your users that withdrawals will now take at least seven days.
```
Comment (anikaraghu) [inline apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md:64]: can remove this line given it was the case before
```suggestion

```

File: apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md [hunk 4]
```diff
@@ -46,25 +45,23 @@ These changes allow anyone to withdraw funds from Base to L1 without having to r
...
 - In addition, the 'DisputeGameFactory' will replace the 'L2OutputOracle' as the new contract for proposing output root statements. This change is part of the broader shift towards the fault proofing system, which is expected to enhance the security and reliability of the platform.
-   
-**If you are in the process of withdrawing your testnet funds from L2 to L1:**
 
-- **Withdrawals _before_ the upgrade in mid-July** will be processed instantaneously.
-- **Withdrawals _during_ or _after_ the fault proofs upgrade** for Base Sepolia will take at least seven days to complete.
+**If you are in the process of withdrawing your funds from L2 to L1:**
 
-If your withdrawal of testnet funds from Base Sepolia to Ethereum Sepolia coincides with the upgrade proceeding in mid-July, you will be required to resubmit your withdrawal.
+- **Withdrawals _during_ or _after_ the fault proofs upgrade** for Base Mainnet will take at least seven days to complete.
```
Comment (anikaraghu) [inline apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md:57]:
```suggestion

- **Withdrawals _before_ the upgrade must wait the 7-day challenge period before finalization.
-  **Withdrawals _during_ or _after_ the fault proofs upgrade** for Base Mainnet will be proven by the Fault Proof system and generally take 7 days to finalize, but could see additional delays if challenged.
```

File: apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md (hunk 2 above)
Comment (wbnns) [reply apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md:4]: c3ebf304

File: apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md (hunk 1 above)
Comment (wbnns) [reply apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md:13]: c3ebf304

File: apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md (hunk 4 above)
Comment (wbnns) [reply apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md:57]: c3ebf304

**Review by anikaraghu — DISMISSED** [review]

**Review by anikaraghu — APPROVED** [review]

**Pull request merged**